optimizely/optimizely.py

```diff
@@ -624,7 +629,7 @@ def get_variation(
 
         return variation_key
 
-    def is_feature_enabled(self, feature_key: str, user_id: str, attributes: Optional[dict[str, Any]] = None) -> bool:
+    def is_feature_enabled(self, feature_key: str, user_id: str, attributes: Optional[UserAttributes] = None) -> bool:
```

Comment:
Can you please confirm, this won't break any code change.

Reply:
This is essentially a no-op wrapper, that gives it a name for the type checker to track. But at runtime the behavior is unchanged.

https://docs.python.org/3/library/typing.html#typing.NewType